spec.html

```diff
@@ -7613,21 +7613,30 @@ <h1>Jobs and Host Operations to Enqueue Jobs</h1>
     <p>Jobs are scheduled for execution by ECMAScript host environments. This specification describes the host hook HostEnqueuePromiseJob to schedule one kind of job; host environments may define additional abstract operations which schedule jobs. Such operations accept a Job Abstract Closure as the parameter and schedule it to be performed at some future time. Their implementations must conform to the following requirements:</p>
 
     <ul>
-      <li>At some future point in time, when there is no running execution context and the execution context stack is empty, the implementation must:
+      <li>At some future point in time, when there is no running execution context and the execution context stack is empty, the implementation must perform the following:
         <ol>
-          <li>Push an execution context onto the execution context stack.</li>
           <li>Perform any implementation-defined preparation steps.</li>
           <li>Call the Abstract Closure.</li>
           <li>Perform any implementation-defined cleanup steps.</li>
-          <li>Pop the previously-pushed execution context from the execution context stack.</li>
+          <li>Assert: The execution context stack is empty.</li>
         </ol>
       </li>
       <li>Only one Job may be actively undergoing evaluation at any point in time.</li>
       <li>Once evaluation of a Job starts, it must run to completion before evaluation of any other Job starts.</li>
       <li>The Abstract Closure must return a normal completion, implementing its own handling of errors.</li>
     </ul>
 
+    <p>At any point in time, an execution is <dfn id="job-preparedtoevaluatecode">prepared to evaluate code</dfn> in _scriptOrModule_ (a Module Record, a Script Record, or *null*) when all of the following conditions are true:</p>
```

Comment:
is "prepared to evaluate code" ever used anywhere?

Reply:
Yes, on line 7650.